src/util/transformArticle.ts

```diff
@@ -26,11 +28,11 @@ function getContent(article) {
   return article.content;
 }
 
-export const transformArticle = (article, locale) => {
+export const transformArticle = (article: GQLArticle, locale: LocaleType) => {
   if (!article) return undefined;
 
   const content = getContent(article);
-  const footNotes = defined(article.metaData.footnotes, []);
+  const footNotes = defined(article?.metaData?.footnotes, []);
```

Comment:
Trenger man egentlig `defined` her?. Defined føles ut som en unødvendig dependency for hele ndla-frontend, da den aldri tar inn en liste med mer enn to elementer. Den brukes kun syv steder.